app/models/participant_declaration.rb

```diff
@@ -80,6 +80,7 @@ class ParticipantDeclaration < ApplicationRecord
   scope :unique_ects,    -> { unique_id.ect }
   scope :unique_mentors, -> { unique_id.mentor }
   scope :unique_uplift,  -> { unique_id.uplift }
+  scope :billable_unique_uplift, -> { unique_id.uplift.where(state: %w[eligible payable paid]) }
```

Comment:
this will work for now, but will soon be obsoleted as we should be working off the line item not the declaration